pubspec.yaml

```diff
@@ -3,14 +3,14 @@
 # found in the LICENSE file.
 
 name: process_runner
-version: 4.1.3
+version: 4.1.4
```

Comment:
It looks like 4.1.3 was never published, so we can leave that version (also that matches the CHANGELOG)

Reply:
Ok, done.